test/services/user.spec.js

```diff
@@ -334,20 +334,20 @@ describe('services', () => {
             // to have something different.  Our next use of Date.now when
             // we add a new entry will then be the most recent time.
             // eslint-disable-next-line prettier/prettier
-            added: new Date(Date.now() - (10 * index)).toISOString()
+            added: new Date(Date.now() - 10 * index).toISOString(),
```

Comment:
I'd prefer not to use the linted code - this has an eslint-disable command that was added to explicitly avoid the linter suggestion.  This is a personal preference, but I feel strongly for this line of code the explicit parentheses should be included.